dephell_archive/_stream.py

```diff
@@ -14,15 +15,35 @@ class ArchiveStream:
     mode = attr.ib(type=str, default='r')
     encoding = attr.ib(type=Optional[str], default=None)
 
-    def exists(self) -> bool:
-        try:
+    def _get_info(self):
+        with suppress(KeyError):
             if hasattr(self.descriptor, 'getmember'):
-                self.descriptor.getmember(str(self.member_path))  # tar
-            else:
-                self.descriptor.getinfo(str(self.member_path))  # zip
```

Comment:
To get this to work for graphviz-0.13.2.zip, at least on py37, I needed to do `self.descriptor.getinfo(str(self.member_path) + '/')`.

This is an existing bug in `ArchiveStream.exists()`, but it is roughly covered by #8 , and will be fixed here I presume, so I wont try to submit a fix.